Account/FxAClient10.swift

```diff
@@ -30,6 +30,18 @@ public struct FxALoginResponse {
     }
 }
 
+public struct FxAccountRemoteError {
+    static let ATTEMPT_TO_OPERATE_ON_AN_UNVERIFIED_ACCOUNT: Int32     = 104
+    static let INVALID_AUTHENTICATION_TOKEN: Int32                    = 110
+    static let ENDPOINT_IS_NO_LONGER_SUPPORTED: Int32                 = 116
+    static let INCORRECT_LOGIN_METHOD_FOR_THIS_ACCOUNT: Int32         = 117
+    static let INCORRECT_KEY_RETRIEVAL_METHOD_FOR_THIS_ACCOUNT: Int32 = 118
+    static let INCORRECT_API_VERSION_FOR_THIS_ACCOUNT: Int32          = 119
+    static let UNKNOWN_DEVICE: Int32                                  = 123
+    static let DEVICE_SESSION_CONFLICT: Int32                         = 124
+    static let UNKNOWN_ERROR: Int32                                   = 999
```

Comment:
Nit: Can we make these more swift-y by making them camel-cased instead of capitalized + underscore?